### DISCUSSION TITLE
Training speed

### DISCUSSION THREAD

**Discussion by ghost** [STVIR/pysot#157] [General]
@lb1100 @StrangerZhang Hi, thanks for your outstanding work, I run the train.py with train.md properly on 2 GPU and 4 GPU respectively, but the speed is almost the same, why? How to modify some super parameters to speed up training？ And  every GPU is GTX1080.

**Comment by ZhiyuanChen** [comment]
What is your batch size?

**Comment by ZhiyuanChen** [comment]
Since no further question is asked, I'm closing it for now.
Please reopen it or create a new issue if you have further question.
